language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/completions/util/QNameRefCompletionUtil.java

```diff
@@ -105,12 +107,20 @@ private static String getAlias(String alias) {
     public static List<Symbol> getModuleContent(PositionedOperationContext context,
                                                 QualifiedNameReferenceNode qNameRef,
                                                 Predicate<Symbol> predicate) {
-        Optional<ModuleSymbol> module = ModuleUtil.searchModuleForAlias(context, 
+        Optional<ModuleSymbol> module = ModuleUtil.searchModuleForAlias(context,
                 QNameRefCompletionUtil.getAlias(qNameRef));
-        return module.map(moduleSymbol -> moduleSymbol.allSymbols().stream()
-                .filter(predicate)
-                .collect(Collectors.toList()))
+        List<Symbol> symbols = module.map(moduleSymbol -> moduleSymbol.allSymbols().stream()
+                        .filter(predicate)
+                        .collect(Collectors.toList()))
```

Comment:
```suggestion
                        .filter(predicate)
                        .collect(Collectors.toList()))
```
We can remove this additional spaces.